src/components/network/PageBlock.vue

```diff
@@ -104,7 +104,13 @@ export default {
           networkId: this.network
         }
       },
+      /* istanbul ignore next */
       update: function(result) {
+        if (!result) {
```

Comment:
this is wrong right? shouldn't this return just an array as the good path is returning `result.validators`

Reply:
yes, looks quite wrong... :eyes:  thx

Reply:
I think we better wait and see if we adopt @iambeone 's solution.

He has being demoing it to me earlier and it really does look far superior to this one.

I am tempted to close this PR quite frankly.